gui/qt/amountedit.py

```diff
@@ -104,3 +104,18 @@ def setAmount(self, amount):
 class BTCkBEdit(BTCAmountEdit):
     def _base_unit(self):
         return BTCAmountEdit._base_unit(self) + '/kB'
+
+class BTCkBEdit2(BTCAmountEdit):
+    def _base_unit(self):
+        return 'sats' + '/kB'
+    def get_amount(self):
+        try:
+            x = Decimal(str(self.text()))
+        except:
+            return None
+        return int( x) if x > 0 else None
+    def setAmount(self, amount):
+        if amount is None:
+            self.setText(" ") # Space forces repaint in case units changed
```

Comment:
When the user edits this, do they get a strange leading space in some circumstances?  If so, it'd be better to explicitly force a repaint, and set empty string.